development-certs/certs/grafana.development.local-key.pem

```diff
@@ -0,0 +1,28 @@
+-----BEGIN PRIVATE KEY-----
```

Comment:
Grafana (https://grafana.com/) is THE best practices standard metrics dashboard tooling.  It includes a fantastic UI for generating dashboards of all kinds of metrics from multiple sources and integrates seemlessly with prometheus.  It is possible to access and embed these graphs and dashboards in other pages, as well as generate graph from multiple heterogenous sources (including, say, our database or something dataset we have).